torch/nn/modules/batchnorm.py

```diff
@@ -347,9 +356,13 @@ def _check_input_dim(self, input):
 
 
 class LazyBatchNorm2d(_LazyBatchNorm):
-    r"""A :class:`torch.nn.BatchNorm2d` module with lazy initialization of
-    the ``num_features`` argument of the :class:`BatchNorm2d` that is inferred
-    from the ``input.size(1)``.
+    r"""A :class:`torch.nn.BatchNorm2d` module with lazy initialization.
+
+    In this module, the `weight` and `bias` are of :class:`torch.nn.UninitializedParameter`
```

Comment:
Same comment as above, I thing the old version is more better as it is not repeating what is already in the general doc and to the point.